extensions/Worker.Extensions.Tables/src/TableInputAttribute.cs

```diff
@@ -43,31 +43,31 @@ public TableInputAttribute(string tableName, string partitionKey, string rowKey)
 
         /// <summary>When binding to a table entity, gets the partition key of the entity.</summary>
         /// <remarks>When binding to an entire table, returns <see langword="null"/>.</remarks>
-        public string? PartitionKey { get; }
```

Comment:
Is the ask to also change this prop to be non-nullable? Or is it just "Take"? If so, could you also update the comments that say "returns null".

Reply:
The original issue only asks to change `int?` to `int` for `Take`, but I noticed that we had `string?` for some properties. Since `string` is a reference type (and therefore intrinsically nullable,) the `?` is unnecessary (it's intended for value types), so I removed them as cleanup.

Reply:
Okay, reading a little further and looking at other worker extensions, it looks like we do want nullable strings in this case: [Nullable reference types and static analysis | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types#nullable-references-and-static-analysis)

Reverting the string? changes.